clients/privacy-center/public/fides-js-components-demo.html

```diff
@@ -119,7 +119,6 @@
           modalLinkId: null,
           privacyCenterUrl: "http://localhost:3000",
           fidesApiUrl: "http://localhost:8080/api/v1",
-          tcfEnabled: false,
```

Comment:
I removed this so that it's not misleading that this does anything right now (sending the option to the `window.Fides.init` function is too late, since by then, we've decided if we're using `fides.js` or `fides-tcf.js`). it's a little confusing to think about, so maybe that means this isn't quite the right approach...